### PR TITLE
CIV-12893 Defendant LiP dashboard notifications: Default judgment issued

### PR DESCRIPTION
**Pull request by German-A-Amoros**
### JIRA link ###
[CIV-12893](https://tools.hmcts.net/jira/browse/CIV-12893)

### Change description ###
Defendant LiP dashboard notifications: Default judgment issued

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by neeta-hmcts — DISMISSED** [review]

File: src/main/services/dashboard/dashboardInterpolationService.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ const setDashboardValues = (claim: Claim, claimId: string, notification?: Dashbo
   const civilMoneyClaimsTelephone = config.get<string>('services.civilMoneyClaims.telephone');
   const civilMoneyClaimsTelephoneWelshSpeaker = config.get<string>('services.civilMoneyClaims.welshspeaker.telephone');
   const cmcCourtEmailId = config.get<string>('services.civilMoneyClaims.courtEmailId');
+  const generalApplicationsUrl = config.get<string>('services.genApps.url');
```
Comment (Azam-Hmcts) [inline src/main/services/dashboard/dashboardInterpolationService.ts:59]: Refactor name of link to "initiateGAUrl" and keep it with "#" or empty

File: config/default.yaml [hunk 2]
```diff
@@ -65,6 +65,8 @@ services:
       telephone: '0300 303 5174'
   applyForCertificate:
     url: 'https://www.gov.uk/government/publications/form-n443-application-for-a-certificate-of-satisfaction-or-cancellation'
+  genApps:
```
Comment (Azam-Hmcts) [inline config/default.yaml:68]: I think configuration is not required

File: config/custom-environment-variables.yaml [hunk 3]
```diff
@@ -41,6 +41,8 @@ services:
     courtEmailId: CIVILMONEYCLAIMS_COURT_EMAIL_ID
     welshspeaker:
       telephone: CIVILMONEYCLAIMS_WELSHSPEAKER_TELEPHONE
+  genApps:
+    url: CIVIL_GENERAL_APPLICATIONS_URL
```
Comment (Azam-Hmcts) [inline config/custom-environment-variables.yaml:45]: Its not required

File: charts/civil-citizen-ui/values.yaml [hunk 4]
```diff
@@ -82,6 +82,7 @@ nodejs:
     CIVILMONEYCLAIMS_COURT_EMAIL_ID: 'ocmcnton@justice.gov.uk'
     APPLY_FOR_CERTIFICATE: https://www.gov.uk/government/publications/form-n443-application-for-a-certificate-of-satisfaction-or-cancellation
     LAUNCH_DARKLY_ENV: '{{ .Values.global.environment }}'
+    CIVIL_GENERAL_APPLICATIONS_URL: http://civil-general-applications-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```
Comment (Azam-Hmcts) [inline charts/civil-citizen-ui/values.yaml:85]: not required

File: src/main/services/dashboard/dashboardInterpolationService.ts [hunk 5]
```diff
@@ -103,6 +103,7 @@ const setDashboardValues = (claim: Claim, claimId: string, notification?: Dashbo
   valuesMap.set('{VIEW_CCJ_REPAYMENT_PLAN_CLAIMANT}', CCJ_REPAYMENT_PLAN_CLAIMANT_URL.replace(':id', claimId));
   valuesMap.set('{VIEW_MEDIATION_SETTLEMENT_AGREEMENT}', VIEW_MEDIATION_SETTLEMENT_AGREEMENT_DOCUMENT.replace(':id', claimId));
   valuesMap.set('{UPLOAD_MEDIATION_DOCUMENTS}', START_MEDIATION_UPLOAD_FILES.replace(':id', claimId));
+  valuesMap.set('{SERVICE_REQUEST_UPDATE}', '#');
```
Comment (Azam-Hmcts) [inline src/main/services/dashboard/dashboardInterpolationService.ts:106]: Is this variable for GA initiation link? Name doesnt say anything about GA ..

**Review by Omaira-Melo-Hmcts — DISMISSED** [review]

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

**Review by krishnanuthalapati — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**